api/services/productService.js

```diff
@@ -0,0 +1,91 @@
+const { productDao } = require('../models');
+const getProductByProductId = async (productId) => {
+    return await productDao.getProductByProductId(productId);
+};
+
+const getProducts = async (limit, offset, sort, minPrice, maxPrice, color, category, brand, productGender) => {
```

Comment:
```suggestion
const getProducts = async (limit = 10, offset = 0, sort, minPrice, maxPrice, color, category, brand, productGender) => {
```
이런식으로 default를 줄 수 있어서 사실 몇몇 파라미터는 꼭 필요한 상황이 아니라면 복잡한 코드를 추가로 작성할 필요가 없긴 합니다 ㅎㅎ